pom.xml

```diff
@@ -133,7 +133,6 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <maven.compiler.fork>true</maven.compiler.fork>
-        <maven.compiler.verbose>true</maven.compiler.verbose>
```

Comment:
By default it is false. You are correct, we can remove this. The new maven compiler plugin generates too much log > 4 Mb and Travis fails the build automatically.